features/step_definitions/networking.rb

```diff
@@ -994,28 +996,26 @@
   network_type = network_operator.network_type(user: admin)
   case network_type
   when "OVNKubernetes"
-    # use -4 to limit output to just `default` interface, fixed in later iproute2 versions
     if v6
        step %Q/I run command on the node's ovnkube pod:/, table("| ip | -6 | route | show | default |")
     else
-       step %Q/I run command on the node's ovnkube pod:/, table("| ip | -4 | route | show | default |")
+       inf_address = admin.cli_exec(:get, resource: "node/#{node_name}", output: "jsonpath={.status.addresses[0].address}")
     end
   when "OpenShiftSDN"
-    step %Q/I run command on the node's sdn pod:/, table("| ip | -4 | route | show | default |")
+       inf_address = admin.cli_exec(:get, resource: "node/#{node_name}", output: "jsonpath={.status.addresses[0].address}")
```

Comment:
Indention issue ?
Also the indention are not consistent. Some of them use 2 whitespace, and some of them use 3, and some of them use 4.

Reply:
Thanks. Just fixed that in vscode